tests/kokkos/kokkos_utils_tests.cpp

```diff
@@ -288,9 +288,14 @@ void test_view_reduction(const Scalar a=Scalar(0.0), const int begin=0, const in
   Kokkos::View<Scalar*> results ("results", 1);
   const auto results_h = Kokkos::create_mirror_view(results);
 
+#ifdef KOKKOS_ENABLE_OPENMP
+  const int n_threads = omp_get_max_threads();
+#else
+  const int n_threads = 1;
+#endif
   // parallel_for over 1 team, i.e. call view_reduction once
   const auto policy =
-    ekat::ExeSpaceUtils<ExeSpace>::get_default_team_policy(1, view_length);
+    ekat::ExeSpaceUtils<ExeSpace>::get_team_policy_force_team_size(1, (UseThreads ? n_threads : 1));
```

Comment:
I don't understand this. On GPU, this only tests a trivial team/vector size (team_size=1, vector_length=1). In that case, the serialized and non-serialized reductions should do the same job, so the test might not see bugs.

I think the policy should have team size larger than 1, and, on Cuda, a vector length larger than 1.